src/main/java/frc/trigon/robot/commands/LEDAutoSetupCommand.java

```diff
@@ -71,6 +75,6 @@ else if (difference > tolerance)
     }
 
     private Pose2d getCurrentRobotPose() {
-        return RobotContainer.POSE_ESTIMATOR.getCurrentPose();
+        return RobotContainer.POSE_ESTIMATOR.getCurrentEstimatedPose();
```

Comment:
this function seems rather useless

Reply:
It cleans it up

Reply:
It gives more ambiguity on where the pose is coming from
No need